### PR TITLE
Add note informing users not to install Flutter and Dart separately

### PR DESCRIPTION
**Pull request by abitofevrything**
Fixes https://github.com/dart-lang/site-www/issues/5938

---

- [x] I’ve reviewed the contributor guide and applied the relevant portions to this PR.
- [x] This PR doesn't contain automatically generated corrections or text (Grammarly, LLMs, and similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn't use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

<details>
  <summary>Contribution guidelines:</summary><br>

  - See our [contributor guide](https://github.com/dart-lang/site-www/blob/main/CONTRIBUTING.md) for general expectations for PRs.
  - Larger or significant changes should be discussed in an issue before creating a PR.
  - Code changes should generally follow the [Dart style guide](https://dart.dev/effective-dart) and use `dart format`.
  - Updates to [code excerpts](https://github.com/dart-lang/site-shared/blob/main/doc/code-excerpts.md) indicated by `<?code-excerpt` need to be updated in their source `.dart` file as well.
</details>

### CODE REVIEW

**Comment by atsansone** [comment]
**question:** This point already exists on this page.

image

Does this not meet your requirement? (Honestly, just asking. Overuse of admonitions is a concern with the docs.)

**Comment by abitofevrything** [comment]
Yes, it is mentioned - but only further down the list than the package manager option. I see a lot of linux users on a flutter support group (FlutterDev on Discord) who see this option and immediately jump to it, skipping over the point about installing flutter underneath.

Another option would therefore be to reorder the list to place the Install Flutter option first, but I also think that having the message stand out in some way (in this case, by being in a tip box) early on in the page can help draw more attention to it as most users of Dart are still currently only using it because of Flutter - and therefore do not need to be visiting the Get Dart page in the first place.

**Comment by atsansone** [comment]
@abitofevrything : Make sense. Let me check the admonitions quick and see if we can get more content above the edge of the first screen.

File: src/content/get-dart/index.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ you need to develop Dart command-line, server, and web apps.
 
 To learn more about the Dart SDK, consult the [Dart SDK overview](/tools/sdk).
 
+:::tip
+If you've installed or plan to [install the Flutter SDK][install-flutter], it
+includes the full Dart SDK and you do not need to install Dart separately.
```
Comment (atsansone) [inline src/content/get-dart/index.md:18]: **suggestion:** 

```suggestion
If you've installed or plan to [install the Flutter SDK][install-flutter], it
includes the full Dart SDK. You don't need to install Dart separately and can skip this guide.
```

**Review by atsansone — APPROVED** [review]

**Pull request merged**